game/src/environment/LevelMap/core/LevelMap.gd

```diff
@@ -21,3 +37,29 @@ export var level_name : String = "core"
 	# $Navigator/NavPolyInstance.set_enabled(false)
 	# $Navigator/NavPolyInstance.queue_free()
 	# $Navigator/NavigationPolygonInstance.set_enabled(true)
+
```

Comment:
hmmm. Im guessing u wanna call on_home_damage to trigger some event? Not sure whats the use case other than reaching 0 hp since u made it so that only an event occurs at 0 hp.  Also, not sure why u dun want home to handle this then u just refer to it instead of creating a variable. guess this works ba but abitttt sus  from me

Reply:
in case we want the UI to show like a counter on planet indicator